appveyor.yml

```diff
@@ -0,0 +1,25 @@
+version: "{build}"
+
+init:
+  - git config --global core.autocrlf input
+
+environment:
+  matrix:
+    # node.js
+    - nodejs_version: "0.10"
+    - nodejs_version: "0.11"
+    # io.js
+    - nodejs_version: "1.0"
```

Comment:
Fun, huh :-)? it is 1.0.4, but its not ready to be used, see - https://iojs.org/faq.html#version

@mikeal do you have a ticket about that? It seems it confuses a lot of people.